.github/workflows/extract-translation-source-files.yml

```diff
@@ -94,7 +94,12 @@ jobs:
             const allGenericRepos = [
               {
                 repo: 'tutor-contrib-aspects',
-                transifex_file_path: 'transifex_input.yaml'
+                transifex_file_path: 'transifex_input.yaml',
+              },
+              {
+                repo: 'openedx-app-ios',
+                transifex_file_path: 'I18N/en.lproj/Localizable.strings',
+                before_extract: 'make translation_requirements',
```

Comment:
@brian-smith-tcril I'd like your opinion here.

The pull request is ready for merge once the iOS PR is merged.